### PR TITLE
End of Part 1.

### PR DESCRIPTION
**Pull request by despinoAltimetrik**

### CODE REVIEW

File: debug/logs/2018-05-11 16-47-49.log [hunk 1]
```diff
@@ -0,0 +1,39 @@
+42.0 APEX_CODE,FINEST;APEX_PROFILING,INFO;CALLOUT,INFO;DB,INFO;SYSTEM,DEBUG;VALIDATION,INFO;VISUALFORCE,INFO;WAVE,INFO;WORKFLOW,INFO
```
Comment (rjulianialtimetrik) [inline debug/logs/2018-05-11 16-47-49.log]: Please remove the debug folder from the repo!!

Comment (despinoAltimetrik) [reply debug/logs/2018-05-11 16-47-49.log]: Debug files were removed two commits afterwards.

File: src/triggers/tasksTrigger.trigger [hunk 2]
```diff
@@ -0,0 +1,32 @@
+trigger tasksTrigger on Task__c (before update, before insert) {
+    for (Task__c t : Trigger.New){
+        String status = t.Status__c;
+        if ((t.Due_Date__c == null) && (t.Type__c == 'Daily')) {
+            t.Due_Date__c = Date.today();
+        }
+        if (status == 'Completed'){
+            String type       = t.Type__c;
+            List<Contact> aux = [SELECT Id, Points__c FROM Contact WHERE Id = :t.Assignee__c];
```
Comment (rjulianialtimetrik) [inline src/triggers/tasksTrigger.trigger]: Please do not use SOQL queries inside **for** loops. 
You should gather the Ids of all contacts you have to update in a Set (from whatever there is inside trigger.New) and then perform an SOQL query to get the actual contact records (possibly storing them inside a map). then you should apply the points calculation just as you did by updating the corresponding contact record that's now contained in the Map. After all that is done, you can perform a single update statement over the Map's values outside any for loop.

Comment (despinoAltimetrik) [reply src/triggers/tasksTrigger.trigger]: Fixed in the next commit.

**Pull request merged**